pages/linux/procs.md

```diff
@@ -0,0 +1,26 @@
+# procs
+
+> procs displays information about the active processes.
+> It can be a used as ps or top alternative
+> More information: <https://github.com/dalance/procs>
+
+- Show information about all the processes:
+
+`procs`
+
+- Show information about process with command or user containg `zsh`:
```

Comment:
I'd rephrase this a little
```suggestion
- Show information about processes containing `zsh` and the commands which started them:
```

Reply:
How about:
Show information about processes, if the commands which started them contain `zsh` :

Reply:
I'm not sure if `if` is the right word here. Let's wait for some other opinions

Reply:
Actually, why not?
```suggestion
- Show information about processes, if the commands which started them contain `zsh`:
```